<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>605</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>605</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>605</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NDS Automation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cmb_buadRate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_Port">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_deviceName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_baudRate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Port">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_deviceNameEntry">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addDevice">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>82</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_deviceTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>422</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>140</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tab_gameSelection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>421</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_RSE">
     <attribute name="title">
      <string>RSE</string>
     </attribute>
     <widget class="QComboBox" name="cmb_rse_category">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_rse_encounter">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_rseCategory">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Category</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_rse_enctounter">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Encounter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_rse_startSequence">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>270</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_rse_BattleRecord">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Battle Record</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_rse_painting">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Painting</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_rse_liveBattery">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Live Battery?</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spb_rse_targetAdvance">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_rse_targetMS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Target (MS)</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_rse_paintingTimer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Painting Time (MS)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spb_rse_paintingTime">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
     </widget>
     <widget class="QTextBrowser" name="txb_rse_preReqs">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>261</width>
        <height>201</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_rse_deviceSel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_rse_deviceSel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Device</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_FRLG">
     <attribute name="title">
      <string>FRLG</string>
     </attribute>
     <widget class="QComboBox" name="cmb_frlg_category">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_frlg_emcounterSel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Encounter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_frlg_enctounterSel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_frlg_category">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Category</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_frlg_buttonSel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_frlg_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Button</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spb_frlg_introTimer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_frlg_introTimer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Intro Timer (MS)</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_frlg_totalTime">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Time (MS)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spb_frlg_totalTime">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spb_frlg_tvFrames">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_frlg_tvFrames">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TV Frames (MS)</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="txb_frlg_preReqs">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>261</width>
        <height>201</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_frlg_deviceSel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_frlg_deviceSel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Device</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_frlg_startSequence">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>270</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chk_frlg_select">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_clearDevices">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Devices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refreshPorts">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh Ports</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
